torch/_decomp/__init__.py

```diff
@@ -53,7 +53,11 @@ def add_op_to_table(aten_op):
                 if op_overload in registry:
                     raise RuntimeError(f"duplicate registrations for {op_overload}")
                 registry[op_overload] = f
-                if register_meta:
+                # TODO: factor this logic into OpOverload or Library API
+                name = op_overload._schema.name
+                if op_overload._schema.overload_name:
+                    name += "." + op_overload._schema.overload_name
```

Comment:
nit: FunctionSchema's pybind binding should really have a .operator_name() like the C++ FunctionSchema so we don't need to do this

Reply:
There's an annoying API mismatch, where some APIs require `foo.default` and others require `foo` (no appended dot).